Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -136,7 +172,7 @@ void Wiimote::Reset()
   m_shake_state = {};
 }
 
-Wiimote::Wiimote(const unsigned int index) : m_index(index)
+Wiimote::Wiimote(const unsigned int index) : m_index(index), m_eeprom_dirty(false)
```

Comment:
You could just initialize this value in the class definition: `bool m_eeprom_dirty = false;`.